main/adapters/repository/database_init.go

```diff
@@ -31,59 +18,18 @@ var create = map[int]string{
 		"public_key BYTEA NOT NULL, "+
 		"signature BYTEA NOT NULL, "+
 		"auth_token VARCHAR(255) NOT NULL, "+
-		"active boolean NOT NULL DEFAULT(TRUE));", PostgresIdentityTableName),
-	PostgresVersion: fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s("+
-		"id VARCHAR(255) NOT NULL PRIMARY KEY, "+
-		"migration_version VARCHAR(255) NOT NULL);", PostgresVersionTableName),
+		"active boolean NOT NULL DEFAULT(TRUE));", IdentityTableName),
```

Comment:
A few thoughts about this:

I propose we drop the reference of `PostgresIdentityTableName`. It is used all over the place, but I think it is better to keep it out of the program logic. The table name is unlikely to be changed, since that would not be backwards compatible. Even if it was changed, we would need to reference the old and the new table name for the respective migration.

By using `identity` instead of `PostgresIdentityTableName` the SQL becomes easier to read, and can be decoupled from the Program code. It also doesn't need `fmt.Sprintf()`-templating anymore.

As another trick: you can decide to use multiline strings, which should also improve readability, since it will get rid of the `"…"+` repetitions:

```go
sqlDDL = `
CREATE TABLE IF NOT EXISTS identity (
	uid VARCHAR(255) NOT NULL PRIMARY KEY,
	private_key BYTEA NOT NULL,
	public_key BYTEA NOT NULL,
	signature BYTEA NOT NULL,
	auth_token VARCHAR(255) NOT NULL,
	active boolean NOT NULL DEFAULT(TRUE)
);
`
```

Reply:
yes, this is a very old PR, sitting here since August. After our conversation about this, I immediately changed it as discussed on a more recent branch. see https://github.com/ubirch/ubirch-client-go/commit/959e9886a59212be8b61ec90312a66c1e5180128